spec/lucky/action_spec.cr

```diff
@@ -101,6 +101,17 @@ describe Lucky::Action do
       Tests::Create.route.should eq Lucky::RouteHelper.new(:post, "/tests")
     end
 
+    it "escapes path params" do
+      Tests::Edit.path("test/id").should eq "/tests/test%2Fid/edit"
+      Tests::Edit.with("test/id").should eq Lucky::RouteHelper.new(:get, "/tests/test%2Fid/edit")
```

Comment:
I think this test needs just these two lines. The rest can be deleted

Reply:
True, rest was repetitive 🏁